src/mangosd/Master.cpp

```diff
@@ -81,10 +81,10 @@ class FreezeDetectorRunnable : public MaNGOS::Runnable
                 uint32 curtime = WorldTimer::getMSTime();
 
                 // normal work
-                if (w_loops != World::m_worldLoopCounter)
+                if (w_loops != static_cast<uint32>(World::m_worldLoopCounter))
```

Comment:
The ```static_cast``` here seems unnecessary, given that ```std::atomic<T>``` has a non-explicit ```operator T()``` - which here becomes ```operator uint32()``` - and ```w_loops``` is also an ```uint32``` variable. 
So that seems like a perfectly normal comparison to me even without the cast. Did you happen to get any compiler warning or so?